lib/ohai/plugins/aix/virtualization.rb

```diff
@@ -17,7 +18,7 @@
 #
 
 Ohai.plugin(:Virtualization) do
-  provides "virtualization"
+  provides "virtualization", "wpars"
```

Comment:
It doesn't look like you're setting wpars as a top-level attribute. Do you mean `"virtualization/wpars"`?